src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.tsx

```diff
@@ -8,22 +8,39 @@ import {
 } from '@fluentui/react';
 import React, { useCallback, useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { createGUID } from '../../../../../Models/Services/Utils';
 import {
+    IDTDLPropertyType,
     IExpressionRangeType,
     IValueRange
 } from '../../../../../Models/Types/Generated/3DScenesConfiguration-v1.0.0';
 import { CardboardList } from '../../../../CardboardList';
 import { ICardboardListItem } from '../../../../CardboardList/CardboardList.types';
-import { transformValueRangesIntoConditions } from '../VisualRuleFormUtility';
+import {
+    isNumericType,
+    transformValueRangesIntoConditions
+} from '../VisualRuleFormUtility';
+import ConditionsCallout from './ConditionsCallout/ConditionsCallout';
 import { getStyles } from './ConditionsList.styles';
 import {
+    CalloutInfo,
+    CalloutInfoType,
     Condition,
     IConditionsListProps,
     IConditionsListStyles,
     IConditionsListStylesProps
 } from './ConditionsList.types';
 
 const LIST_KEY = 'cb-visual-rule-conditions-list';
+const getDefaultCondition = (type: IDTDLPropertyType): IValueRange => ({
```

Comment:
we actually have a bunch of these "default" initializers somewhere in the app. It might be in ViewerConfigUtility? Let's add this one there as well to have them all together.